client/.eslintcache

```diff
@@ -0,0 +1 @@
ningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"48","messages":"49","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"50","messages":"51","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"52","messages":"53","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"54","messages":"55","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"56","messages":"57","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"58","messages":"59","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"60","messages":"61","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"62","messages":"63","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"64","messages":"65","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"66","messages":"67","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"68","messages":"69","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/index.js",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/App.js",["70"],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/serviceWorker.js",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Home.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/withAuth.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Secret.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Login.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Header.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/EyeCatcher.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Gallery.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Logo.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Navigation.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/Story.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/SmallStory.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/StoryBody.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/StoryFooter.jsx",[],"/home/hackyourfuture/Downloads/hack-my-future/Hear-My-Story/client/src/StoryHeader.jsx",[],{"ruleId":"71","severity":1,"message":"72","line":2,"column":10,"nodeType":"73","messageId":"74","endLine":2,"endColumn":14},"no-unused-vars","'Link' is defined but never used.","Identifier","unusedVar"]
```

Comment:
Would remove this file should not be committed
https://github.com/Sayed94h/Hear-My-Story/pull/21/files Sayed added it to be ignored in the build fix PR